.changes/bundler-object-custom-sign-command.md

```diff
@@ -0,0 +1,6 @@
+---
+"tauri-bundler": patch:breaking
+"tauri-utils": patch:breaking
+---
+
+Custom sign command for windows executables is now with object instead of string to support whitespaces in the command path and arguments.
```

Comment:
I wonder if it is worth supporting both formats, string is more concise for simple use cases and object is better for white space paths and such

Reply:
cc @lucasfernog

Reply:
supporting both formats is what we do :) use an enum with serde(untagged)

Reply:
@anatawa12 sorry for the inconvenience, could you add support for both? and add a note in docs that users should use the object format if their binary or arguments have spaces.

Reply:
OK.
Should support quote in string notation for whitespaces? or use old simple parser with split(' ')?

Reply:
use old simple parser, and just add a note in docs

Reply:
I added string form support in 53e285a1e9b248fbd6c821a8b975c028aedee302.

Since string format is only for simpler config file, I added parsing logic in cli module and bundler module have single object form as input.